### ISSUE TITLE
Proposals for Unit 5: Getting Started with Accessibility

### ISSUE THREAD

**Issue by gernus** [w3c/wai-curricula#126]
Sustainability of the accessibility of websites should be more prominent

Methods/Strategies to ensure the accessibility of websites over the time.

**Comment by daniel-montalvo** [comment]
Hi Gerhard,

Thanks for your input.

One of the teaching resources for this module is [Planning and Managing Web Accessibility](https://www.w3.org/WAI/planning-and-managing/), which includes [Sustain](https://www.w3.org/WAI/planning-and-managing/sustain/) as well as many other planning and managing strategies.

I propose the following tweak so that sustainability  is identified from the very top of the module.

## Introduction

Current wording: "* Introduce different strategies on how to plan, apply, and manage accessibility."

Suggested change: "* Introduce different strategies on how to plan, apply, manage, and sustain accessibility."

Would this change address the concern? Do you think further changes to learning outcomes, teaching ideas or ideas to assess knowledge are needed to make sustainability even clearer?


Thanks.

**Comment by gernus** [comment]
Hi Daniel!

I think, your suggested change is perfect.

Cheers
Gerhard
